test/test_bort.F90

```diff
@@ -25,12 +28,57 @@ program test_bort
   print *, 'Testing ', sub_name, ' case ', test_case
 
   ! Run the test for the subroutine and test case.
-  if (sub_name .eq. 'bort') then
+  if (sub_name .eq. 'adn30') then
+     if (test_case .eq. '1') then
+        char_short = adn30(0, 5)
+     elseif (test_case .eq. '2') then
+        char_30 = adn30(0, 9)
+     endif
+  elseif (sub_name .eq. 'bort') then
      if (test_case .eq. '1') then
         call bort('goodbye!')
-     else if (test_case .eq. '2') then
+     endif
+  elseif (sub_name .eq. 'bort2') then
+     if (test_case .eq. '1') then
         call bort2('goodbye!', 'goodbye again!')
      endif
+  elseif (sub_name .eq. 'bvers') then
+     if (test_case .eq. '1') then
+        call bvers(char_short)
+     endif
+  elseif (sub_name .eq. 'cmpmsg') then
+     if (test_case .eq. '1') then
+        call cmpmsg('W')
+     endif
+  elseif (sub_name .eq. 'writdx') then
+     if (test_case .eq. '1') then
+        call writdx(0, 0, 0)
+     endif
+  elseif (sub_name .eq. 'writlc') then
+     if (test_case .eq. '1') then
+        call writlc(0, 0, 0, 0)
+     endif
```

Comment:
writlc has only 3 call arguments, and all but the first one are character strings, not integers.

Reply:
Fixed.